index.php

```diff
@@ -1,10 +1,41 @@
 <?php
 
-$db = new PDO(
-    'mysql:host=192.168.20.20, dbname=Guitars',
+$db = new PDO
+(
+    'mysql:host=192.168.20.20; dbname=Guitars',
     'root',
     ''
 );
 
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
+/**
+ *
+ * Returns all desired item values for all items.
+ * Where many images are stored for an item, only the first added (MAIN image) is returned
+ *
+ * @param $database
```

Comment:
What is the database? Should mention it's the connection established using PDO or similar

Reply:
have added more relevant information to DocBlock that details what input parameters and outputs should be